it/src/main/java/com/google/cloud/teleport/it/launcher/PipelineLauncher.java

```diff
@@ -173,10 +173,10 @@ public static Builder builder(TestName testName) {
     }
 
     /** Builder for the {@link LaunchConfig}. */
-    public static final class Builder {
+    public static final class Builder<T> {
       private final String jobName;
       private final String specPath;
-      private final Map<String, String> environment;
+      private final Map<String, T> environment;
```

Comment:
nit: This feels more like a `Map<String, Object>` (to allow any value type) rather than a specific `T`. I feel <T> without specifying a type will reach the same outcome, but it may raise some raw type warnings.

Reply:
I agree, changed it to `Map<String, Object>`